### PR TITLE
Updated in/readme.md with current readme 

### PR DESCRIPTION
**Pull request by sagarhedaoo**
### All Submissions:

- [Yes ] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/reflex-dev/reflex/blob/main/CONTRIBUTING.md) file?
- [Yes ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/reflex-dev/reflex/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

### New Feature Submission:

- [Yes ] Does your submission pass the tests? 
- [Yes ] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [Yes ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [N/A ] Have you written new tests for your core changes, as applicable?
- [Yes ] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.

    b. Describe your changes.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

**Comment by sagarhedaoo** [comment]
closes #3315

File: docs/in/README.md [hunk 1]
```diff
@@ -85,44 +102,47 @@ class State(rx.State):
...
+            rx.button("Generate Image", on_click=State.get_image, width="25em"),
             rx.cond(
-                State.complete,
-                     rx.image(
-                         src=State.image_url,
-                         height="25em",
-                         width="25em",
-                    )
+                State.processing,
+                rx.chakra.circular_progress(is_indeterminate=True),
```
Comment (Lendemor) [inline docs/in/README.md:124]: Latest readme shouldn't have the `rx.chakra.circular_progress` anymore

**Review by Lendemor — CHANGES_REQUESTED** [review]
thanks for helping.

I think for the code you've used the version prior to latest (just before some of the code change in the example)

File: docs/in/README.md (hunk 1 above)
Comment (sagarhedaoo) [reply docs/in/README.md]: Hi! Apologies, Just changed the code

**Review by picklelo — APPROVED** [review]

**Pull request merged**